main/src/io/github/iltotore/iron/RefinedTypeOps.scala

```diff
@@ -48,7 +48,7 @@ object RefinedTypeOps:
      */
     type FinalType = T
 
-trait RefinedTypeOpsImpl[A, C, T]:
+trait RefinedTypeOpsImpl[A, C, T](using rtc: RuntimeConstraint[A, C]):
```

Comment:
Looks good to me. There is a very slight breaking change (aka requiring the constraint to be available at creation but I think it was more an incoherence that needed to be fixed than a feature).

Reply:
Yeah I agree 👍. Also since the code is inlined, I believe this change doesn't cause any binary incompatibility issues.

Would you like that I clean up this PR and mark it as ready for review?

Reply:
If you think it is ready, sure!